libs/stache/src/lib/modules/blockquote/blockquote.component.scss

```diff
@@ -1,18 +1,28 @@
 @use '../../styles/_variables' as vars;
 
 .stache-blockquote {
-  border-left: 4px solid vars.$stache-background-color-primary-dark;
-  font-size: vars.$stache-font-size-base * 1.12;
-  color: vars.$stache-text-color-deemphasized;
+  border-left: var(--sky-border-width-accent, 4px) solid
+    var(--sky-color-border-info, vars.$stache-background-color-primary-dark);
+  font-size: calc(vars.$stache-font-size-base * 1.12);
```

Comment:
do we have a token we could use here, or do none really apply?

Reply:
Just missed it - updated with a token